_js/main.js

```diff
@@ -3,6 +3,9 @@ import SearchController from './lib/SearchController'
 import HelpController from './lib/HelpController'
 import $ from 'jquery'
 import URI from 'urijs'
+import Prism from 'prismjs'
+
+Prism.highlightAll()
```

Comment:
This will highlight only on initial load. If you use links in the site to navigate to a page that had code block, highlight is not run.

Reply:
Is there an alternative approach that would avoid that issue, or what do you suggest?

I wouldn't consider this a show-stopper, as the result is readable without highlighting (as it is now), and syntax highlighting can be restored by reloading the page in the browser.